src/main/resources/config/liquibase/changelog/20240829165533_added_additionalInfo_to_Company.xml

```diff
@@ -0,0 +1,18 @@
+<databaseChangeLog
+      xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+      xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+      xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+          http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd
+          http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+          http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-latest.xsd">
+
+  <changeSet author="john" id="20240829165533">
```

Comment:
We seem to use jhipster in the author field in the past which is wrong. Let's use oncokb going forward.

Reply:
so `author="oncokb"`?

Reply:
yeah